VISUALIZERS/PLOTLY/LINE/MULTIPLE_TIME_LINES.py

```diff
@@ -0,0 +1,18 @@
+from flojoy import flojoy, DataContainer
+import plotly.graph_objects as go
+
+
+@flojoy
+def MULTIPLE_TIME_LINE(v, params):
+    dc_input = v[0]
+    if dc_input.type == "ordered_pair":
+        Numberoflines = dc_input.x  # If dc.x is just a variable
```

Comment:
`number_of_lines`

variables in python almost always follow this convention

Reply:
yes, deleted this file because not possible to plot timed lines for the moment